content/command-line/concepts/bash/bash.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Bash'
+Description: 'Bash is a Unix-based shell and language used for tasks like system administration and software testing.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Computer Science'
+Tags:
+  - 'Bash/Shell'
+  - 'Command Line'
+  - 'Linux'
+  - 'Unix'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'paths/computer-science'
+---
+
+The Bourne Again Shell, or **[Bash](https://www.gnu.org/software/bash/manual/bash.html)**, is a [Unix](https://www.codecademy.com/resources/docs/general/unix)-based shell program and language used for a multitude of purposes including system administration and software testing.
+
+Developed by Brian Fox in 1989, Bash was released as part of the GNU Project to replace the original Bourne Shell. It was also one of the first programs ported to [Linux](https://www.codecademy.com/resources/docs/open-source/linux).
```

Comment:
Concern: I notice Linux is in open-source. Should this be moved to `general/operating-systems` (General reorganization ref) instead?

Reply:
Yes, I agree. I can remove this link and replace once the new link for `general/operating-systems` exists. I was actually hoping to do this for macOS, Windows, and other operating system concepts in the future. 😄